src/main/java/net/sacredlabyrinth/phaed/simpleclans/listeners/SCEntityListener.java

```diff
@@ -78,6 +79,18 @@ public void onEntityDeath(EntityDeathEvent event)
 
             if (attacker != null && victim != null)
             {
+            	if (SimpleClans.getInstance().getSettingsManager().isDenySameIPKills()) 
+            	{
+            		if (attacker.getAddress().getHostString().equals(victim.getAddress().getHostString())) 
```

Comment:
This can be moved up into the second if-statement with an `&&` to avoid unnecessary nesting.